### ISSUE TITLE
Default nbsite proposal

### ISSUE THREAD

**Issue by jbednar** [pyviz/nbsite#49]
nbsite can handle a variety of different site configurations, but I'd like to propose a default for most PyViz sites to use .  I'll use datashader as an example, which currently looks like:

![image](https://user-images.githubusercontent.com/1695496/40327748-a59e558e-5d09-11e8-87ea-8ac2c1cedede.png)
...
![image](https://user-images.githubusercontent.com/1695496/40327757-acda341c-5d09-11e8-91e4-c3d2a3464a68.png)

The layout currently looks jumbled and haphazard to me, and I'd like to address that. In particular:

- The little logo in the top left looks a bit odd when there is a big logo, but I think the big logo is important for branding.
- The little logo isn't using the wordmark, which again looks odd next to the big one.
- The sidebar is mostly empty
- The Twitter feed looks crammed in
- There's a lot of space taken at the bottom by the page footer
- There are at least three ways to navigate, all of which look like they are different things (top bar, side bar, page footer).  Two or three places to do the same thing is convenient; three different-seeming ways to do the same thing is confusing.

**Comment by jbednar** [comment]
Here's one possible mockup addressing most of the points above, but I'm still thinking about it:

![image](https://user-images.githubusercontent.com/1695496/40331495-31e5692c-5d16-11e8-9374-472e28d97c4e.png)

**Comment by jbednar** [comment]
Note that the above approach is for a site where the sidebar is always visible; it would need a different approach if it weren't.  

Also note that the top bar should presumably stay visible on scrolling, as in http://numba.pydata.org/numba-doc/0.38.0/index.html ...

**Comment by jlstevens** [comment]
I agree datashader.org is a bit of a mess, especially with that huge logo.

My reference here is holoviews.org...
![image](https://user-images.githubusercontent.com/890576/40361255-9c9d27fa-5dc0-11e8-8991-dd23aa16db8d.png)

and bokeh.org...

![image](https://user-images.githubusercontent.com/890576/40361290-b9e29412-5dc0-11e8-82a7-ef99a4272935.png)

which I think look fine.

**Comment by jlstevens** [comment]
Philipp pointed out that the holoplot site looks fine with the logo (and I agree):

![image](https://user-images.githubusercontent.com/890576/40361451-4af8539c-5dc1-11e8-86dd-1174f03713fb.png)
